src/components/newsletter-preview/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * WordPress dependencies
+ */
+import { BlockPreview } from '@wordpress/block-editor';
+import { Fragment, useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const NewsletterPreview = ( { meta = {}, ...props } ) => {
+	const ELEMENT_ID = useMemo( () => `preview-${ Math.round( Math.random() * 1000 ) }`, [] );
```

Comment:
@adekbadek This is a neat solution to create randomized IDs, but would it be simpler to pass the `usedLayout.ID` as a prop to `<NewsletterPreview />`? We wouldn't need a memoization then, either.

Reply:
Good idea, but the id will not be unique if there are two previews of the same layout visible at once ;) 
This situation happens in layout selection modal – there's the little preview in the left column, and the large preview in the right column.

Reply:
I get it now, thanks for explaining. Looks good to me, then!